source/ProtonPack/ProtonPack.ino

```diff
@@ -3114,77 +3158,77 @@ void wandStoppedFiring() {
 }
 
 void wandStopFiringSounds() {
-  if(b_wand_firing == true) {
-    // Firing sounds.
-    switch(FIRING_MODE) {
-      case PROTON:
-      default:
-        if(i_mode_year == 1989) {
-          stopEffect(S_GB2_FIRE_START);
-        }
-        else {
-          stopEffect(S_GB1_FIRE_START);
-        }
+  ms_firing_stop_sound_delay.stop();
 
-        if(i_mode_year == 2021) {
-          stopEffect(S_AFTERLIFE_FIRE_START);
-        }
-        stopEffect(S_GB1_FIRE_LOOP);
+  // Firing sounds.
+  switch(FIRING_MODE) {
+    case PROTON:
+    default:
+      if(i_mode_year == 1989) {
+        stopEffect(S_GB2_FIRE_START);
         stopEffect(S_GB2_FIRE_LOOP);
+      }
+      else {
+        stopEffect(S_GB1_FIRE_START);
+        stopEffect(S_GB1_FIRE_LOOP);
+      }
 
-        stopEffect(S_FIRING_LOOP_GB1);
-        stopEffect(S_GB1_FIRE_START_HIGH_POWER);
-        stopEffect(S_GB1_FIRE_HIGH_POWER_LOOP);
-        stopEffect(S_FIRE_START_SPARK);
-        stopEffect(S_FIRE_START);
-      break;
+      if(i_mode_year == 2021) {
+        stopEffect(S_AFTERLIFE_FIRE_START);
+      }
 
-      case SLIME:
-        stopEffect(S_SLIME_START);
-        stopEffect(S_SLIME_LOOP);
-        stopEffect(S_SLIME_END);
-      break;
+      stopEffect(S_FIRING_LOOP_GB1);
+      stopEffect(S_GB1_FIRE_START_HIGH_POWER);
+      stopEffect(S_GB1_FIRE_HIGH_POWER_LOOP);
+      //stopEffect(S_FIRE_START_SPARK);
```

Comment:
Again, just checking if commented for testing or to be removed?

Reply:
As mentioned in a previous comment, I was lessening the burden on the serial buffer queue. Particularly on sound effects which are very short in nature and can play out instead of sending a serial command to stop it. For now I would leave it commented out but in the code.